lib/srv/db/redis/test.go

```diff
@@ -117,7 +115,7 @@ func TestServerPassword(password string) TestServerOption {
 }
 
 // NewTestServer returns a new instance of a test Redis server.
-func NewTestServer(t testing.TB, config common.TestServerConfig, opts ...TestServerOption) (*TestServer, error) {
+func NewTestServer(config common.TestServerConfig, opts ...TestServerOption) (*TestServer, error) {
```

Comment:
we don't need to link these test servers either for production. i can look into that separately.